api/simplyrets-openapi.yaml

```diff
@@ -2300,6 +2425,129 @@ components:
+          items:
+            type: string
+          example: ["srmls", "abcmls", "xyzmls"]
+          description: |
+              A list of vendor IDs available to these API
+              credentials. These vendor IDs can be used in requests to
+              `/properties`, `/openhouses`, etc.
+        updates:
+          nullable: false
+          type: object
+          example:
+            simplyrets: "2020-08-01T11:06:18.916879378Z"
+          description: |
+            A key/value map of vendor IDs with the timestamp of the
+            last succesful sync.
+        endpoints:
+          nullable: false
+          type: array
+          items:
+            type: string
+          example: ["/properties", "/openhouses"]
+          description: |
+              A list of API endpoints accesible with these credentials.
+        accepts:
+          nullable: false
+          type: array
+          items:
+            type: string
+          example: ["application/json", "application/vnd.simplyrets-v0.1+json"]
+          description: |
+              A list of valid media types available to these API credentials.
+                See [versioning API requests with media types](/topics/versioning-api-requests-with-media-types).
+    PropertiesOptionsResponse:
+      type: object
+      description: Response of a successful OPTIONS request to /properties
+      properties:
+        expires:
+          nullable: false
+          type: string
+          format: date-time
+          example: "2020-01-02T11:06:18.916879378Z"
+          description: |
+              Timestamp when the information in this response should
+              be considered stale.
+        lastUpdate:
+          nullable: false
+          type: string
+          format: date-time
+          example: "2020-01-01T11:06:18.916879378Z"
+          description: |
+              Timestamp when this feed was last synced with the data
+              source successfully.
+        fields:
+          nullable: true
+          type: object
+          description: |
+              An object containing field names, and all known values
+              for that field in this feed.
+          properties:
+            status:
+              nullable: true
+              type: array
+              items:
+                type: string
+              example: ["Active", "Pending", "Closed"]
+              description: A list of `statuses` available for this feed.
+            cities:
+              nullable: true
+              type: array
+              items:
+                type: string
+              example: ["Houston", "Austin", "Dallas"]
+              description: A list of `cities` found in this feed.
+            counties:
+              nullable: true
+              type: array
+              items:
+                type: string
+              example: ["Houston", "Austin", "Dallas"]
+              description: A list of `counties` found in this feed.
+            features:
```

Comment:
I think we should consider removing features from the response. Can you check who originally made the request to have it in the response?

Reply:
replied in slack